### DISCUSSION TITLE
Tailwind 2.2 with Sage 10 by Roots

### DISCUSSION THREAD

**Discussion by grzesiek1owline** [tailwindlabs/tailwindcss#4961] [Help]
Hi, i used sage 10 with tailwind.
I have a problem with browsersync. When i have many images in 'resources' browsersync works veryyy slow.
When i dont have images, i have a problem with styles injection. DOM elements has new classes, but not their definitions.
I must refresh page manually..

Sage 10 [laravel mix + webpack]
Taiwlind 2.2.4 jit mode
Ubuntu 18.04

**Comment by simonswiss** [comment]
Hey, can you please share a working code repo that demonstrates the problem?

Issue with styles and manual page refresh needed is very likely an issue with your webpack setup - be sure to go through the troubleshooting section of the Tailwind CSS JIT mode docs:

https://tailwindcss.com/docs/just-in-time-mode#troubleshooting
